lib/database/adapters/RunAdapter.js

```diff
@@ -209,6 +225,17 @@ class RunAdapter {
         entityObject.flpRoles = flpRoles ? flpRoles.map(this.flpRoleAdapter.toEntity) : [];
         entityObject.lhcPeriod = lhcPeriod ? this.lhcPeriodAdapter.toEntity(lhcPeriod) : lhcPeriod;
 
+        entityObject.avgInelasticInteractionRate = avgInelasticInteractionRate;
+        entityObject.inelasticInteractionRateAtStart = inelasticInteractionRateAtStart;
+        entityObject.inelasticInteractionRateAtMid = inelasticInteractionRateAtMid;
+        entityObject.inelasticInteractionRateAtEnd = inelasticInteractionRateAtEnd;
+        if (lhcFill && avgInelasticInteractionRate !== null) {
+            entityObject.muInelasticInteractionRate =
+                avgInelasticInteractionRate / (extractNumberOfCollidingLhcBunchCrossings(lhcFill.fillingSchemeName) * LHC_REVOLUTION);
```

Comment:
Do not hesitate to use temporary variables to make the code more readable, for example `numberOfCollidingLhcBunchCrossings`

Reply:
done